dependency_injection/services.php

```diff
@@ -40,6 +40,12 @@
 return static function (ContainerConfigurator $container): void {
     $projectDir = dirname(__DIR__);
 
+    $parameters = $container->parameters();
+
+    // Default secret, just in case
+    $parameters->set('env(APP_SECRET)', uniqid('', true));
+    $parameters->set('kernel.secret', env('APP_SECRET'));
```

Comment:
There is a `glpicrypt.key` file in the config dir that is generated during installation/update. We should rely on it, I guess.
See `GLPIKey` class.